### DISCUSSION TITLE
Unable to use 'create static web app' in Technical Onboarding

### DISCUSSION THREAD

**Discussion by Jayati-Wajire** [microsoft/SATechnicalOnboarding#70] [Questions]
I am an Alpha MLSA and claimed my Azure credits successfully and tried to create a static web app for completing my Technical Onboarding Process, but this came up my way. Please, provide some guidelines.

![Troubleshoot1](https://github.com/microsoft/SATechnicalOnboarding/assets/109218479/b7ab7097-c8cc-4528-8e3e-5c294a0cb2a2)

**Comment by AnthonyByansi** [comment]
Hello, @Jayati-Wajire,

It seems you are encountering an error due to the absence of a resource group for your Azure resource, specifically the `static web app`. To resolve this issue, it's essential to create a dedicated resource group for your deployment. You can achieve this by navigating to `portal.azure.com`. If you require any further assistance or clarification on the process, feel free to ask.

**Reply by iveambassador** [reply]
I have the same problem even though the resource group already exists.

**Reply by Jayati-Wajire** [reply]
Problem solved. Thank you for your guidance!

**Reply by AnthonyByansi** [reply]
Happy you gat that resolved, @Jayati-Wajire!

**Reply by AnthonyByansi** [reply]
@Jayati-Wajire Also be sure to make this as answered!

**Comment by shelar1423** [comment]
even im facing the same issue

**Comment by mfarooqk47** [comment]
I am having the same issue.

**Comment by mfarooqk47** [comment]
![image](https://github.com/microsoft/SATechnicalOnboarding/assets/75903910/86aeb2f2-eedf-4518-b7dd-d3d93c5f499c)
please help with this issue. it is persistently poping up

**Comment by AnthonyByansi** [comment]
Hi @Jayati-Wajire, @iveambassador, @shelar1423, and @mfarooqk47,

I'd like to highlight that the creation of resource groups and individual services via the Azure Extension in `Vscode` may not be fully supported. For a more stable approach, I recommend provisioning these services directly from the Azure portal. For clarification, please access `portal.azure.com` to create the resource group along with the corresponding services. This ensures a smoother and more reliable provisioning process.
